app/services/fraud_indicator_service.rb

```diff
@@ -24,7 +24,7 @@ def fraud_suspected?
   def acceptable_timezone_strings
     @acceptable_timezone_strings ||= begin
       us_timezones = ActiveSupport::TimeZone.us_zones.map { |tz| [tz.name, tz.tzinfo.name].uniq }.flatten.freeze
-      overrides = IceNine.deep_freeze!(YAML.load_file("lib/timezone_overrides.yml"))
+      overrides = ["America/Indianapolis"]
```

Comment:
removed the overrides file in anticipation of using the timezone table -- left this implementation here so that i dont break tests.